src/utils/promises.js

```diff
@@ -1,7 +1,8 @@
 /* @flow */
-import type { PromiseType } from '../types';
 
-export const promiseCompletionPercentage = (promiseList: Array<PromiseType>): number => {
+export const promiseCompletionPercentage = <T: { fulfilled: boolean }>(
```

Comment:
https://stackoverflow.com/questions/44155934/flowtype-array-of-object-type-does-not-allow-extra-properties